code/game/objects/structures/mirror.dm

```diff
@@ -23,7 +24,7 @@
 	if(broken || !Adjacent(user))
 		return
 
-	if(ishuman(user))
+	if(ishuman(user) && !magical)
```

Comment:
can't you just not call parent in magic window?

Reply:
it is supposed to call its parent's parent, so, not calling its parent doesn't look a good idea.